src/radioButtonGroup/readMe.md

```diff
@@ -0,0 +1,46 @@
+## RadioButtonGroup
+
+The base RadioButtonGroup component.
+
+### Properties
+| Name | Type | Default | Description |
+|:-----|:-----|:-----|:-----|
+| `options` | `Array` | &nbsp; | Options to render checkboxes for. |
+| `onClick` | `Function` | &nbsp; | Callback function to be called when the radio button is clicked |
+| `inline` | `boolean` | `false` | Prop to define the alignment of radio button group |
+
+### Theme
+
+| Name     | Description|
+|:---------|:-----------|
+| `radio-group`   | Class used for the root element.|
+| `customized-radio`   | Class used for the radio button element.|
+| `each-check`   | Class used for the checkbox element.|
```

Comment:
Re write the docs since it is mixed with check and radio. Every name should be related only to radio

Reply:
Docs typo fixed.